extensions/EXT_color_buffer_half_float/extension.xml

```diff
@@ -37,14 +37,16 @@
         Renderbuffers can be created in these formats. These and textures
         created with <code>type = HALF_FLOAT_OES</code>, which will have one
         of these internal formats, can be attached to framebuffer object color
-        attachments for rendering.</p>
+        attachments for rendering. <code>RGBA16F</code> textures are guaranteed
+	to be color-renderable. <code>RGB16F</code> textures are color-renderable
+        if <code>checkFramebufferStatus()</code> returns <code>FRAMEBUFFER_COMPLETE</code>.
```

Comment:
I think the most correct way to do this is to make both RGB16F and RGBA16F formats color-renderable, but only guarantee not to be FRAMEBUFFER_UNSUPPORTED for RGBA16F. If the implementation doesn't support RGB16F, it would generate FRAMEBUFFER_UNSUPPORTED.

I do think we should deprecate RGB16F, since it's not actually renderable in ES3+ even with extensions.

Reply:
Revised. We can discuss RGB16F removal but I will leave that out of this pull request.